src/coreComponents/mesh/generators/VTKMeshGenerator.cpp

```diff
@@ -510,13 +522,113 @@ std::vector< T > collectUniqueValues( std::vector< T > const & data )
 }
 
 /**
- * @brief Get the GEOSX element type
+ * @brief Identify the type of the polyhedron
+ *
+ * @param cell The vtk cell VTK_POLYHEDRON
+ * @return geosx::ElementType The geosx element type associated to VTK_POLYHEDRON
+ */
+geosx::ElementType polyhedronType( vtkCell * const cell )
```

Comment:
```suggestion
geosx::ElementType buildGeosxPolyhedronType( vtkCell * const cell )
```
Please prefix with a verb.